extensions/ql-vscode/src/view/common/SuggestBox/SuggestBox.tsx

```diff
@@ -0,0 +1,236 @@
+import type { FormEvent, ReactNode } from "react";
+import { useCallback, useEffect, useMemo, useRef, useState } from "react";
+import {
+  autoUpdate,
+  flip,
+  FloatingFocusManager,
+  FloatingPortal,
+  size,
+  useDismiss,
+  useFloating,
+  useFocus,
+  useInteractions,
+  useListNavigation,
+  useRole,
+} from "@floating-ui/react";
+import { styled } from "styled-components";
+import { VSCodeTextField } from "@vscode/webview-ui-toolkit/react";
+import type { Option } from "./options";
+import { findMatchingOptions } from "./options";
+import { SuggestBoxItem } from "./SuggestBoxItem";
+
+const Input = styled(VSCodeTextField)`
+  width: 430px;
+
+  font-family: var(--vscode-editor-font-family);
+`;
```

Comment:
Should this have a specified background colour too? When I tested in Storybook, the light themes looked like this: 
![image](https://github.com/github/vscode-codeql/assets/42641846/35771e49-9b69-4b6d-954e-51ef41483f92)

Reply:
That seems to be a bug in the Storybook theme, it does work correctly when rendered in VS Code. You can see the same behavior in for example the [Variant analysis story](http://localhost:6006/?path=/story/variant-analysis-variant-analysis--full-example&globals=vscodeTheme:light).

Reply:
Ah, I see! That's annoying, but good to know that it's fine in VS Code 👌🏽